.platform.app.yaml

```diff
@@ -140,7 +140,12 @@ hooks:
 
         # optional: run migration automatically with deploy
         bin/console database:migrate --all
-        bin/console cache:clear
+
+        # run cache clear commands, on grid environments this should run in the deploy hook,
+        # on dedicated generation 2, it will run as part of the pre_start hook (to be set up by Platform.sh support)
+        if [ "$PLATFORM_REGISTRY_NUMBER" ]; then
+            sh prestart_cacheclear.sh
```

Comment:
But I guess you still want to clear the redis 🤔 maybe add a cache pool

Reply:
That makes sense, we can do a `bin/console cache:pool:clear` in the deploy hook
Do you know which pool(s) we need to clear then?

Reply:
```
bin/console cache:pool:clear cache.object
```